lib/bulk_processor/back_end/dynosaur/tasks.rb

```diff
@@ -0,0 +1,2 @@
+require_relative 'process_csv_task'
+require_relative 'split_csv_task'
```

Comment:
The difference is that a client using the `dynosaur` back-end needs to explicitly require the rake tasks in `Rakefile`. Having this file allows us to update the tasks as need be without having the client change the require line in the app.